document-store/src/main/java/org/hypertrace/core/documentstore/ClosableIterator.java

```diff
@@ -0,0 +1,6 @@
+package org.hypertrace.core.documentstore;
+
+import java.io.Closeable;
+import java.util.Iterator;
+
+public interface ClosableIterator<E> extends Iterator<E>, Closeable {}
```

Comment:
```suggestion
public interface CloseableIterator<E> extends Iterator<E>, Closeable {}
```
Should match jdk interface name

Reply:
Done